### PR TITLE
[wip] changed DomainRequest and RecordRequest models

### PR DESCRIPTION
**Pull request by ghost**
Fields changed from 'target_' to '', eg. 'target_name' to 'name'.
This is becuase of new concept.
The new concept is to always create DomainRequest (or RecordRequest) and then
(if auto_accept condition is met) write DomainRequest (or RecordRequest) to
Domain (or RecordRequest).

Conflicts:
    powerdns/admin.py

### CODE REVIEW

File: dnsaas/settings.py [hunk 1]
```diff
@@ -227,3 +227,8 @@ def __getitem__(self, item):
         from settings_local import *  # noqa
     except ImportError:
         pass
+
+# list of domains for which RecordsRequests are instantly saved to Records
+# model
+# eg FORCE_AUTO_ACCEPT_DOMAINS = ['domain.pl', 'domain2.pl']
+FORCE_AUTO_ACCEPT_DOMAINS = []
```
Comment (mkurek) [inline dnsaas/settings.py]: couldn't this be attached to Domain model (as a field, ex. `auto_accept`)?

Comment (ghost) [inline dnsaas/settings.py]: Great, I'll do that.

Comment (dominikkowalski) [inline dnsaas/settings.py]: FORCE_AUTO_ACCEPT_DOMAINS = Unrestricted (field in the domain configuration)

Unrestricted -> Can users that are not owners of this domain add recordsto it without owner's permission?

**Comment by mkurek** [comment]
since there is #79, this one could be closed right?

**Comment by ghost** [comment]
yup

**Pull request closed without merging**